api/rosetta/construction.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package rosetta
+
+const (
+	// FlowSignatureAlgorithm denotes the signing algorithm supported by Flow.
+	FlowSignatureAlgorithm = "ecdsa"
+)
+
+// TODO: We could consider moving Data and Construction APIs to separate packages.
```

Comment:
Touched on this with Brendan, I think we should consider separating these two into separate packages.
However, I'd do this after the construction API is complete, to minimize the number of moving pieces.

Reply:
I'm not sure how I would feel about doing this right now tbh. Maybe in the future, but at the moment switching from a monorepo to having multiple ones to maintain might be a bit annoying. It means adding one feature might require multiple PRs, that documentation would be spread between multiple repos, etc.

I think there is some merit to splitting a project between multiple repositories and it could be the case for the Rosetta part of the project, but I don't think it would outweigh the downsides as it is right now 🤔 WDYT?